assets/style.css

```diff
@@ -0,0 +1,140 @@
+body{
+    background-color: #f5f5f5;
+}
+header{
+    position: fixed;
+    background-color: white;
+    top: 0px;
+    left: 0px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    padding: 0px;/*20px;*/
+    z-index: 7;
+    justify-content: space-between;
+    
+}
+
+.sea{
+    display: flex;
+    width: 35%;
+    padding: 0px 200px 0px 200px ;
+    
+}
+.sea button{
+    border: 1px solid grey;
+}
+.icons{
+    display: flex;
+    padding: 5px;
+    margin-right: 35px;
+}
+
+.ib{
+    width:30px;
+    padding: 10px;
+}
+    
+
+#mainlogo{
+    width: 70px;
+    height: 30px;
+    padding: 0px;
+    padding-left: 50px;
+}
+
+#bell{
+    width: 15px;
+    height: 15px;
+}
+#application{
+    width: 15px;
+    height: 15px;
+}
+#create{
+    width: 15px;
+    height: 15px;
+}
+
+aside{
+    position: absolute;
+    right: 50px;
+    top: 70px;
+    width: 30%;
+    
+}
+.search{
+    width: 100%;
+    height: 30px;
+}
+
+main{
+    position: absolute;
+    top: 80px;
+    width: 50%;
+    margin-left: 100px;
+}
+
+/*#youtube{
+    
+}*/
+.user{
+    width: 35px;
+    height: 35px;
+    border-radius: 50%;
+    
+}
+#subscribe{
```

Comment:
You repeated this selector again on line 108. Please group them together on the next projects.